### PR TITLE
Add Alerting V2 config to monitoring

### PR DESCRIPTION
**Pull request by westlywright**
Exposes the basic alerting on the monitoring chart. 

Allows users to enable/disable alerting manager.
Exposes selecting a previously created alert manager secret from the required namespace if exists.
If not, allows user to create the default secret for alert manager. Will also create the default namespace in this case.

rancher/dashboard#1015
rancher/dashboard#1088

@vincent99 I left this is draft mode because I wanted to get your opinion on the CruSecret component. Originally I designed this to replace multi-line text input with the yaml editor but I backed this out as I started to feel weird about assuming what they'd be adding to the secret data, it doesn't have to all yaml strings and we dont do this on the secret create page. If it doesn't make sense to include the yaml editor I'll remove that whole component and move the logic up one level. 

~Also I am seeing an error from the backend when trying to create the secret with the attach the template file data from the chart as indicated by rancher/dashboard#1088. I am investigating whether this is FE/BE and need to sync with arvind.~

### CODE REVIEW

File: chart/monitoring/index.vue [hunk 1]
```diff
@@ -80,16 +99,117 @@ export default {
...
+      }
+    },
+
     async fetchDeps() {
-      const storageClasses = await this.$store.dispatch('cluster/findAll', { type: STORAGE_CLASS });
-      const pvcs = await this.$store.dispatch('cluster/findAll', { type: PVC });
+      const { $store } = this;
+      const storageClasses = await $store.dispatch('cluster/findAll', { type: STORAGE_CLASS });
+      const pvcs = await $store.dispatch('cluster/findAll', { type: PVC });
+      const secrets = await $store.dispatch('cluster/findAll', { type: SECRET });
```
Comment (vincent99) [inline chart/monitoring/index.vue:154]: Fetch these in parallel with all/allHash instead of waiting for each other

Comment (westlywright) [reply chart/monitoring/index.vue]: updated this block

**Comment by aiyengar2** [comment]
@vincent99 creating a default and modifying it after the fact like you mentioned sounds good to me.

**Comment by westlywright** [comment]
@vincent99 Updated with changes for your feedback. Dropped the secret modification. I didn't implement Cody's component for additional secrets obviously but I did swap it out for labeled select with multi which I think works well until Cody's component can be updated.

**Pull request merged**